### PR TITLE
Implement secure trips and get rid of libiconv

### PR DESCRIPTION
**Pull request by ComSecNinja**
Here's a summary of what I've done: 
* Fix typos
* Implement secure tripcodes
* Add simple test to secure tripcodes
* Use `golang.org/x/text/transform` instead of qiniu's wrapper for `libiconv` which didn't work for me

You might want to re-insert wercker for your project and replace "ComSecNinja" with "aquilax" in 
* README.md
* tripcode.go

### CODE REVIEW

File: tripcode_test.go [hunk 1]
```diff
@@ -11,25 +11,33 @@ func TestGenerateSalt(t *testing.T) {
...
+
 func TestTripcode(t *testing.T) {
 	cases := map[string]string{
 		"asd":        "TAPy3blMsc",
 		"adasd":      "IOuORdzMKw",
 		"!@#$%^&*()": "BpZUCmJAIQ",
 		"f}E":        "oUBoOTrysY",
-		"©":          "",
-		"訛":          "c8eDXvwFLQ",
+		"©":          "hEp4vbueZo", // Should be nothing?
```
Comment (aquilax) [inline tripcode_test.go]: `©` is supposed to be ignored. Check https://github.com/KenanY/tripcode/blob/master/test/index.js#L69 and must return empty string.

Comment (ComSecNinja) [inline tripcode_test.go]: Is there any other source for this claim? Both [minichan](https://minichan.org/triptest?name=%23%C2%A9) and [desktopthread](http://desktopthread.com/tripcode.php) tell me that a lone `©` should equal to a trip `cPUZU5OGFs` without variations. Not that I yet know how this is achieved.

Comment (aquilax) [inline tripcode_test.go]: Good question. I checked several other implementations and they return `cPUZU5OGFs` as well, so you're right on this one.

**Comment by aquilax** [comment]
Awesome work :+1:

**Pull request merged**